packages/postgresql.rb

```diff
@@ -3,11 +3,11 @@
 class Postgresql < Package
   description 'PostgreSQL is a powerful, open source object-relational database system.'
   homepage 'https://www.postgresql.org'
-  version "17.0-#{CREW_ICU_VER}"
+  version "16.4-#{CREW_ICU_VER}"
```

Comment:
Downgrade?

Reply:
Isn't the current version in Chromebrew 16.4?